README.rst

```diff
@@ -2,6 +2,11 @@
 Django Request Cache
 ====================
 
+.. image:: https://badge.fury.io/py/django-request-cache.svg
+    :target: https://badge.fury.io/py/django-request-cache
+.. image:: https://github.com/anexia/django-request-cache/actions/workflows/test.yml/badge.svg?branch=master
```

Comment:
Can you please also add a badge for `codecov` and use `main` as the default branch instead of `master`?

Reply:
Changed :+1: